### PR TITLE
Gdr 2199

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2199

## Why was it changed?
Replace `matrix` by `combination`

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: R/convert_mae_se_assay_to_dt.R [hunk 1]
```diff
@@ -113,7 +113,7 @@ convert_se_assay_to_dt <- function(se,
   checkmate::assert_string(assay_name)
   
   object <- assays(se)[[assay_name]]
-  checkmate::assert_true(inherits(object, "BumpyDataFrameMatrix") || inherits(object, "matrix"))
+  checkmate::assertClass(object, "BumpyDataFrameMatrix")
```
Comment (j-smola) [inline R/convert_mae_se_assay_to_dt.R:116]: Are you sure we don't want to have `matrix` as se?
take a look: 
- https://github.com/gdrplatform/gDRutils/blob/GDR-2199/tests/testthat/test-convert_mae_se_assay_to_dt.R#L9
- https://github.com/gdrplatform/gDRutils/blob/GDR-2199/tests/testthat/test-convert_mae_se_assay_to_dt.R#L95

**Review by j-smola — COMMENTED** [review]
Shouldn't be also `matrix` change into `combination` in examples and test data?
```
> mae <- get_synthetic_data("finalMAE_combo_matrix_small")
> mae
A MultiAssayExperiment object of 2 listed
 experiments with user-defined names and respective classes.
 Containing an ExperimentList class object of length 2:
 [1] matrix: SummarizedExperiment with 6 rows and 2 columns
 [2] single-agent: SummarizedExperiment with 5 rows and 2 columns
Functionality:
 experiments() - obtain the ExperimentList instance
 colData() - the primary/phenotype DataFrame
 sampleMap() - the sample coordination DataFrame
 `$`, `[`, `[[` - extract colData columns, subset, or experiment
 *Format() - convert into a long or wide DataFrame
 assays() - convert ExperimentList to a SimpleList of matrices
 exportClass() - save data to flat files
```
since 
```
> gDRutils::get_experiment_groups()
$`single-agent`
  single-agent    co-dilution 
"single-agent"  "co-dilution" 

$combination
[1] "combination"
```
Right?

Please, additionally update `gDRutils.Rmd`

**Review by j-smola — APPROVED** [review]
Please, update package version
* gDRtestData (>= 1.1.3)
* gDRstyle(>=1.1.1)

PS: It seems that `magrittr` may be removed from dependencies list.

File: R/convert_mae_se_assay_to_dt.R (hunk 1 above)
Comment (bczech) [reply R/convert_mae_se_assay_to_dt.R]: fixed

**Pull request merged**